qubesadmin/tests/mock_app.py

```diff
@@ -496,6 +566,39 @@ def __init__(self, qapp: QubesTest, dev_class: str,
         self.backend_vm = backend_vm
         self.attached = attached
 
+        self.update_calls()
+
+    def device_string(self):
+        if self.dev_class == 'block':
+            port, d_id = self.dev_id.split(":", 1)
+            return f"{self.dev_id} device_id='{d_id}' port_id='{port}' " \
+                f"devclass='block' backend_domain='{self.backend_vm}' " \
+                f"serial='root/test.img' manufacturer='{self.description}'  " \
+                "interfaces='b******'\n"
+        elif self.dev_class == 'pci':
+            " vendor='Intel Corporation' interfaces='p0c0500'"
```

Comment:
leftover string, I wonder if pylint complains about stuff like this

Other than this (and assuming CI happy), it's okay now